### DISCUSSION TITLE
Issue Template broken (404)

### DISCUSSION THREAD

**Discussion by KevinMichaelSchindler** [etsy/open-api#1127] [Q&A] [answered]
Heyho :)

Is it intended that the issue template is removed? Should the Q&A discussion section be used instead?

Here the link to the broken page: https://github.com/etsy/open-api/issues/new/choose

Best regards,

Kevin

**Comment by Gareth064** [comment, marked as answer]
Yes, only discussions are allowed now.

This is basically a community support discussion and not a place to raise issues where you want Etsy to respond.

You can search discussions to see if other members have had the same issue. If not the. Raise a new discussion to see if any community members can help. If we can't then you have email Etsy developers for help.

**Reply by KevinMichaelSchindler** [reply]
Alright, great thanks 👍
